app/views/cms/articles/new.html.haml

```diff
@@ -17,11 +17,11 @@
     = form.label :body
     = form.text_area :body
   %p
-    = form.label :author
-    = form.text_field :author
+    = form.hidden_field :user_id, value: "1"
```

Comment:
Why are we using hardcoded `value:"1"` here?

Reply:
fixed